TrafficSimulator/Assets/TrafficLights/TrafficLight.cs

```diff
@@ -0,0 +1,105 @@
+using UnityEngine;
+
+enum State{RED, TOGO, TOSTOP, GREEN};
+
+public class TrafficLight : MonoBehaviour
+{
+    public GameObject RedLight;
```

Comment:
The light gameobjects are not connected to the prefab and currently needs to be manualy inserted or there is an error